### ISSUE TITLE
[MU4 Issue] Navigation through palettes using keyboard no longer possible

### ISSUE THREAD

**Issue by Phil-TA** [musescore/MuseScore#14925]
**Describe the bug**
MuseScore 3 provides a way of quickly navigating through palettes using the keyboard. This is no longer possible with MuseScore 4.

**To Reproduce**
1. Select a palette (for example "Clefs") with a left mouse click
2. Hit key "D"

**Expected behavior**
The Palettes panel scrolls (jumps) to the "Dynamics" palette

**Actual behavior**
A D note is entered into the score!
(or if a key other than A, B, C, D, E, F, or G was pressed then nothing happens at all).

**Screenshots**
N/A

**Platform information**
OS: Windows 10 Version 2009, Arch.: x86_64, MuseScore version (64-bit): 4.0.0-223330401, revision: github-musescore-musescore-3277328

**Additional context**
N/A

**Comment by MarcSabatella** [comment]
Keyboard navigation in palettes definitely works, using the cursor keys.  But you need to first put your keyboard focus in the palette - clicking does not do that.  Instead, use standard accessibility shortcuts like Tab and F6 (with or without Shift) to move keyboard focus.  Or use the palette search shortcut Ctrl+F9.  Then you can type the first couple of characters of "Dynamics" and you will see all dynamics displayed.

**Comment by Phil-TA** [comment]
Thanks Mark. In v3.6, clicking a palette name gave it both mouse **and** keyboard focus - from there, hitting a key would then "cycle" through the open palettes beginning with that letter. This was very handy for swift navigation to less commonly used elements (I have a custom palette set up for the more common elements). I appreciate using palettes' initial letters is a non-standard accessibility shortcut, but it was very handy for speedier workflow. However, I'm sure the Palette search feature will speed things up for me - I've not used that before and it works well.

**Comment by MarcSabatella** [comment]
Indeed, I didn't know that was even possible; I don't think it's documented, and might just be kind of an accident that it works (like if the palettes are implemented using list widgets).  I am not sure if reimplementing that behavior would break any of the new MU4 accessibility workflows, but if it doesn't, certainly that behavior could potentially be reinstated.  I just wanted to clarify, the palettes *are* keyboard-accessible in the ways that were actually intended to work.

**Comment by jmwmusic** [comment]
First letter navigation in palettes and other lists would significantly increase the speed and reduce the number of arrow presses to move through large lists, especially when you have to pause to hear each one to see if it is what you want.  See also issue #16508.
